multichain-testing/Makefile

```
@@ -178,9 +181,9 @@ agoricNames.chain:
 ##
 # Using ymax
 open-with-usdn: deploy-ymax usdc-available ./scripts/ymax-tool.ts
-	./scripts/ymax-tool.ts 1.01
+	(MNEMONIC=$(TRADER1_MNEMONIC) ./scripts/ymax-tool.ts 1.03)
```

I'm never sure about shell quoting. `TRADER1_MNEMONIC` is more than 1 word. Does that work without quoting as below?

```suggestion
	(MNEMONIC="$(TRADER1_MNEMONIC)" ./scripts/ymax-tool.ts 1.03)
```

---

I think wrapping variable values in quotes ("$TRADER1_MNEMONIC") can help avoid issues if they contain spaces or special characters.

---

It definitely contains spaces. But it's reported to work. Hence my question

---

@dckc it works without quotes.

---

looks like it works because the value of `TRADER1_MNEMONIC` includes quotes.

Using this `Makefile`:

```make
M = "1 2 3"

go:
	MM=$(M) python -c "from os import environ; print(repr(environ['MM']))"
```

I get...

```console
$ make
MM="1 2 3" python -c "from os import environ; print(repr(environ['MM']))"
'1 2 3'
```

but without quotes in the value of M:

```make
M = 1 2 3

go:
	MM=$(M) python -c "from os import environ; print(repr(environ['MM']))"
```

I get:

```console
$ make
MM=1 2 3 python -c "from os import environ; print(repr(environ['MM']))"
/bin/sh: 1: 2: not found
make: *** [Makefile:4: go] Error 127
```